fix_vector_store_rag.py

```
@@ -0,0 +1,45 @@
+"""Script to fix the Vector Store RAG.json file."""
+
+import json
+from pathlib import Path
+
+file_path = "src/backend/base/langflow/initial_setup/starter_projects/Vector Store RAG.json"
```

Pylint (3.3.7)

[convention] Constant name "file_path" doesn't conform to UPPER_CASE naming style

(C0103)